src/constructor.io/models/Quizzes/QuestionOption.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+/**
+ * Constructor.io QuestionOption Response
+ */
+namespace Constructorio_NET.Models
+{
+    public class QuestionOption
+    {
+        [JsonProperty("id")]
+        public int Id { get; set; }
+
+        [JsonProperty("value")]
+        public string Value { get; set; }
+
+        [JsonProperty("attribute")]
+        public OptionAttribute Attribute { get; set; }
+    }
```

Comment:
I believe the `multiple` and `single` question types have an [image property in options](https://docs.constructor.io/rest_api/quiz/using_quizzes/#multiple-select).